packages/typespec-ts/test/azureModularIntegration/generated/client/namespace/tspconfig.yaml

```diff
@@ -0,0 +1,16 @@
+emit:
+  - "@azure-tools/typespec-ts"
+options:
+  "@azure-tools/typespec-ts":
+    "emitter-output-dir": "{project-root}"
```

Comment:
Could you try if the namesapce is udpated after spec changed with option `enableModelNamespace` and `enableOperationGroup` ?